### ISSUE TITLE
Gephi 0.9.7: Concentric Layout Plugin Missing

### ISSUE THREAD

**Issue by shubhopalit** [gephi/gephi-plugins#281]
The concentric Layout Plugin seems to be missing from the latest Gephi 0.9.7 version. Is there something I am missing? Thanks in advance.

**Comment by mbastian** [comment]
No, we haven't received any updates about this plugin from its [author](https://github.com/vijeshm/my-gephi-plugins/tree/master/ConcentricLayout). I guess it was never migrated to the 0.9+ version.

Just wondering if [this plugin](https://gephi.org/plugins/#/plugin/circularlayout) isn't at least partially doing the same thing. Not sure.
